Marlin/src/pins/stm32f1/pins_BTT_SKR_MINI_E3_V2_0.h

```diff
@@ -0,0 +1,55 @@
+/**
+ * Marlin 3D Printer Firmware
+ * Copyright (c) 2020 MarlinFirmware [https://github.com/MarlinFirmware/Marlin]
+ *
+ * Based on Sprinter and grbl.
+ * Copyright (c) 2011 Camiel Gubbels / Erik van der Zalm
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+#pragma once
+
+#define SKR_MINI_E3_V2
+#include "pins_BTT_SKR_MINI_E3.h"
+
+#define BOARD_INFO_NAME "BIGTREE SKR Mini E3 V2.0"
+
+// Release PA13/PA14 (led, usb control) from SWD pins
+#define DISABLE_DEBUG
+
+// Onboard I2C EEPROM
+#define I2C_EEPROM
+#define MARLIN_EEPROM_SIZE 0x1000                 // 4KB
```

Comment:
There was mention of this board having an AT24C23 on board. If this is a 32Kb chip, you might define the EEPROM size as `0x8000‬`. This would allow the full EEPROM to be used and allow a lot more large meshes to be saved.

Reply:
Would increasing the EEPROM size potentially fix the issue where it doesn't get past the Marlin boot logo? I would assume not, but I can test this if needed. Perhaps I should try a smaller EEPROM size like 2kb, just to see if it's any better?

Reply:
This board has AT24C32 EEPROM. It is 4kb (32K bit) chip https://ww1.microchip.com/downloads/en/DeviceDoc/doc0336.pdf

Reply:
> This board has AT24C32 EEPROM. It is 4kb (32K bit) chip

I wonder why BigTreeTech doesn’t use the full 4kb in their example firmware.

Reply:
This is what they have declared:
```
#define I2C_EEPROM  // On board AT24C32
#define E2END       (0xFFF) // 4KB
```